assets/src/edit-story/components/canvas/pagenav/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+
+const Wrapper = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	height: 100%;
+	color:  ${ ( { theme } ) => theme.colors.fg.v1 };
+`;
+
+const Space = styled.div`
+	flex: 0 1 ${ ( { direction } ) => 'next' === direction ? 60 : 115 }px;
+`;
+
+const IconStyle = css`
+	border: solid ${ ( { theme } ) => theme.colors.fg.v1 };
+	border-width: 0 1px 1px 0;
+	display: inline-block;
+	padding: 3px;
+	cursor: pointer;
+	opacity: .25;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	color: inherit;
+	height: 25px;
+    width: 25px;
+	&:hover {
+		color: inherit;
+		opacity: 1;
+	}
+`;
+
+const IconNext = styled.i`
+	${ IconStyle }
+	transform: rotate(-45deg);
+    -webkit-transform: rotate(-45deg);
```

Comment:
I'm pretty sure `styled-components`takes care of this. Also, we're only supporting _last-few-versions_ of popular browsers, all supporting `transform`.

Also, there's an indentation problem (also elsewhere in the file).

Reply:
Updated in d7f5e34.